### DISCUSSION TITLE
Is read-only access ok for Harvesters?

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#1944] [Q&A] [answered]
I tracked the file operation in farming. There are only read operations but no write operations. In order to separate plotting and farming, is it ok to give harvesters read-only access to plot files?

If the answer is no, will the write operation change the size of the file or just cover some of the bytes in the file? It is really matters because my storage is kind of like tape storage (but has faster random access than actual tape). If the write operation will change the size of the file, it will not possible to store on this kind of drive.

**Comment by hoffmang9** [comment, marked as answer]
Setting completed plots to read only is a prudent and supported safety measure.

**Reply by cccat6** [reply]
Oh, thanks! It is good news that I could use these read-only drives with a big space of storage.
